src/components/common/verify-code/verify-code-controller.ts

```diff
@@ -100,12 +100,10 @@ export function verifyCodePost(
           );
         if (accountInterventionsResponse.data.blocked) {
           nextEvent = USER_JOURNEY_EVENTS.PERMANENTLY_BLOCKED_INTERVENTION;
-        }
-        if (accountInterventionsResponse.data.temporarilySuspended) {
-          nextEvent = USER_JOURNEY_EVENTS.TEMPORARILY_BLOCKED_INTERVENTION;
-        }
-        if (accountInterventionsResponse.data.passwordResetRequired) {
+        } else if (accountInterventionsResponse.data.passwordResetRequired) {
           nextEvent = USER_JOURNEY_EVENTS.PASSWORD_RESET_INTERVENTION;
+        } else if (accountInterventionsResponse.data.temporarilySuspended) {
```

Comment:
Can you confirm intervention takes priority over suspended accounts?

Reply:
Password reset intervention takes priority over temporary suspension as all accounts with password reset required are temporarily suspended, though not all temporarily suspended accounts have password reset required. Meaning a user must have password reset intervention removed first before then re-running the check to see if temporary suspensions are applied for other reasons